.github/CODEOWNERS

```diff
@@ -78,18 +79,15 @@
 /src/plugins/ui_actions/ @elastic/kibana-app-services
 /src/plugins/index_pattern_field_editor @elastic/kibana-app-services
 /src/plugins/screenshot_mode @elastic/kibana-app-services
+/src/plugins/bfetch/ @elastic/kibana-app-services
+/src/plugins/index_pattern_management/ @elastic/kibana-app-services
+/src/plugins/inspector/ @elastic/kibana-app-services
 /x-pack/examples/ui_actions_enhanced_examples/ @elastic/kibana-app-services
 /x-pack/plugins/data_enhanced/ @elastic/kibana-app-services
 /x-pack/plugins/embeddable_enhanced/ @elastic/kibana-app-services
 /x-pack/plugins/ui_actions_enhanced/ @elastic/kibana-app-services
 /x-pack/plugins/runtime_fields @elastic/kibana-app-services
 /x-pack/test/search_sessions_integration/ @elastic/kibana-app-services
-#CC# /src/plugins/bfetch/ @elastic/kibana-app-services
```

Comment:
@elastic/kibana-qa, in my understanding changing from `CC` to regular code owner shouldn't break code coverage tooling?

Reply:
Correct.  
`#CC#` means it's used for Code Coverage team assignment but does trigger reviews on PRs.
Everything not marked `#CC#` does both Code Coverage and PR reviews.